packages/core/helper-plugin/src/hooks/useFocusWhenNavigate.ts

```diff
@@ -1,8 +1,17 @@
 import { useEffect } from 'react';
 
-const useFocusWhenNavigate = (selector = 'main', dependencies = []) => {
+type DependencyList = ReadonlyArray<unknown>;
+interface UseFocusWhenNavigateProps {
+  selector?: string;
+  dependencies?: DependencyList;
+}
```

Comment:
I agree, i'm not sure there's any benefit, if i'm honest I would actually import `DependencyList` from `react` because if the types change there we can react accordingly (pun not intended)

Reply:
@joshuaellis  I agree import from React is more secure.

Reply:
Thanks for reminder